recipes/acorns/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "acorns" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/deshanadesai/autodiff/archive/v0.0.1.tar.gz
+  sha256: 64c1de7798d247b3987b886b84da367d189a7debe4670c9f9d687a67c4eb7ce0
+
+build:
+  number: 0
+  skip: True  # [win or py<35]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pycparser
+
+test:
+  imports:
+    - src
+    - tests
```

Comment:
@DeshanaDesai these are artifacts of incomplete packaging choices. I don't believe you want your package to be "import src" and having the tests outside of the module will  cause headaches.

If you are following the "src" layout for packages please take a look at PyPA recommendations about it.

Reply:
Yes, completely agree. I have changed the "src" to "acorns" in the next version. But yet to be committed. Thanks for checking!